.github/workflows/ci-rust.yml

```diff
@@ -2,7 +2,19 @@ name: CI Rust
 
 on:
   workflow_call:
+    inputs:
+      run-wasm-tests:
+        description: Run wasm test (mostly in platform crates)
```

Comment:
I think the "mostly" comment is not needed because we are describing the input, which is either run wasm tests or not.

Reply:
Done